app/models/ems_refresh/link_inventory.rb

```diff
@@ -26,7 +26,7 @@ def link_ems_inventory(ems, target, prev_relats, new_relats)
     do_disconnect = target.kind_of?(ExtManagementSystem)
 
     # Do the Folders to Folders relationships
-    update_relats(:folders_to_folders, prev_relats, new_relats) do |f|
+    update_relats(:folders_to_folders, prev_relats, new_relats, disconnect) do |f|
```

Comment:
Could we just change [this](https://github.com/pkliczewski/manageiq/blob/86c2dd0106505581d0eaac7f8724139affd3b50d/app/models/ems_refresh/link_inventory.rb#L26) to set do_disconnect if disconnect is false?

Reply:
I will check